makefile

```diff
@@ -114,6 +118,19 @@ test-backend-local: backend/api-service/src/api/openapi.yaml
 	cd backend/api-service/src && FIRESTORE_EMULATOR_HOST=localhost:9090 go test -tags=emulator -v
 	docker stop firestore-emulator
 
+FIREBASE_SA=$(shell gcloud --project=$(PROJECT_ID) iam service-accounts list --filter="displayName=firebase-adminsdk" --format="value(email)")
+test-istio-auth-local:
+	gcloud --project=$(PROJECT_ID) iam service-accounts keys create --iam-account=$(FIREBASE_SA) \
+		/tmp/istio-auth-test-key.json
+	cd istio-auth && API_KEY=$$(grep apiKey ../webui/firebaseConfig.ts | cut -d "'" -f2) \
+		HOST_IP=$$(kubectl -n $(ISTIO_INGRESS_NAMESPACE) get service $(ISTIO_INGRESS_SERVICE) -o jsonpath='{.status.loadBalancer.ingress[0].ip}') \
+		GOOGLE_APPLICATION_CREDENTIALS=/tmp/istio-auth-test-key.json \
+		go test -v || touch /tmp/istio-auth-test.failed
+	gcloud --project=$(PROJECT_ID) -q iam service-accounts keys delete --iam-account=$(FIREBASE_SA) \
+		$$(jq -r .private_key_id /tmp/istio-auth-test-key.json)
```

Comment:
I think we need to document in the prerequisites that `jq` must be installed, or add a `makefile` rule that installs it.

Reply:
Hmm, should it be added to the README or can we just check via `which jq` and print a message that it needs to be installed for this target?

Reply:
It's probably fine to add a `jq` target that just check if it exists on the `PATH` and prints an error otherwise.

Reply:
Adding a `makefile` target sounds a bit tricky considering the potentially different developer platforms.

Reply:
I'll add the target that checks to make sure it exists

Reply:
Done.